apps/web/src/containers/TradingContest/points-dashboard/my-breakdown.tsx

```diff
@@ -3,15 +3,19 @@ import { THEME_VARIANTS } from '@notional-finance/util';
 import { useMyBreakdownTable } from './hooks';
 import { useNotionalTheme } from '@notional-finance/styles';
 import { ThemeProvider } from '@mui/material';
+import { observer } from 'mobx-react-lite';
+import { usePointsDashboardStore } from './init-points-dashboard';
 
 export const MyBreakdown = () => {
```

Comment:
When you export the default, we should remove the export here so that we don't accidentally import the wrong one function